### ISSUE TITLE
Following 'Get started' results in 'Invalid format version detected - Expected:[ 1 ] Found: [ 2 ]' error 

### ISSUE THREAD

**Issue by KucokDobra** [JacopoPatroclo/nx-react-native-expo#5]
My version of node is v14.15.3.

When I follow instructions in 'Get started':
- create empty NX workspace
- add nx-react-native-expo plugin
- generate an app
- run

I get the error

![image](https://user-images.githubusercontent.com/54781170/103418024-6f9d8c00-4b8d-11eb-9cf9-39d072314865.png)

And indeeded version of workspace.json is 2. If I change version in workspace to 1 I get another error.
![image](https://user-images.githubusercontent.com/54781170/103418116-92c83b80-4b8d-11eb-9d6d-8ef03b0e5cf6.png)

**Comment by github-anis-snoussi** [comment]
same issue here.
(node version 14.15.0)

**Comment by benjaminkoetting** [comment]
same for node version v12.20.0, it depends on the version 2 inside workflow.json file

**Comment by npwork** [comment]
Node: v15.5.0
OS: Mac OS 11.1 (20C69)
"nx-react-native-expo": "0.0.8",
NPM: 6.14.11


Same here

**Comment by npwork** [comment]
**Workaround**: comment following lines in
`node_modules/nx-react-native-expo/node_modules/@angular-devkit/core/src/workspace/json/reader.js:31:15`

```js
/*if (formatVersion !== 1) {
       throw new Error(`Invalid format version detected - Expected:[ 1 ] Found: [ ${formatVersion} ]`);
}*/
```

**Comment by JacopoPatroclo** [comment]
Good day to everyone and sorry for the delay. I've pushed a fix [here](https://github.com/JacopoPatroclo/nx-react-native-expo/pull/7) and it should work properly with version 1 and 2 (using a d). If anyone was kind enough to give me some feedbacks, it will be appreciated. Comment directly under the pull request. In any case, I will publish a new version with this fix during the next week

**Comment by JacopoPatroclo** [comment]
Fix in new version 0.0.9
